client/src/Picks.tsx

```diff
@@ -1004,6 +1005,48 @@ function ForceBeatmapsetUpdate({ nomination }: { nomination: INomination }) {
   );
 }
 
+interface DescriptionDifferenceProps {
+  oldDescription: string;
+  newDescription: string;
+}
+
+function DescriptionDifference({
+  oldDescription,
+  newDescription,
+}: DescriptionDifferenceProps): JSX.Element {
```

Comment:
return type probably not useful
i usually omit them on components cuz 1. this is stricter than is actually required 2. react itself will cry anyway if u dont return reactnode